### ISSUE TITLE
AWS: terraform destroy does not work with Terraform 0.15.0

### ISSUE THREAD

**Issue by cmd-ntrf** [ComputeCanada/magic_castle#161]
There is a regression introduced in Terraform 0.15.0 that hinders the ability to destroy a cluster when using.

When calling `terraform destroy` with AWS, the following error is displayed:
```
│ Error: Invalid AWS Region:
│   on ./aws/infrastructure.tf line 2, in provider "aws":
│    2: provider "aws" {
```

This is a bug in Terraform. Until a new released is published, the solution is to not use Terraform 0.15.0 with Magic Castle AWS.

**Comment by cmd-ntrf** [comment]
Fixed in 0.15.1.

**Comment by cmd-ntrf** [comment]
I'll leave this opened until we update the README.
